src/main/java/seedu/address/ui/DoublePersonCard.java

```diff
@@ -60,34 +67,70 @@ public DoublePersonCard(Person seller, Person buyer) {
         address.setText(seller.getAddress().value);
         email.setText(seller.getEmail().value);
 
-        StringJoiner propertyJoiner = new StringJoiner("\n");
-        seller.getProperties().stream().map(Property::toString).forEach(propertyJoiner::add);
-        property.setText(propertyJoiner.toString());
+        if (seller.getUserType().isBuyer()) {
+            tagBuyer.setText(seller.getUserType().value);
+            tagSeller.setVisible(false);
+            tagSeller.setManaged(false);
+        } else {
+            tagSeller.setText(seller.getUserType().value);
+            tagBuyer.setVisible(false);
+            tagBuyer.setManaged(false);
+        }
+
+        if (!seller.getFavourite().isUnfavourited()) {
+            favourite.setText(seller.getFavourite().toString());
+        } else {
+            favourite.setManaged(false);
+        }
+
+        if (seller.getProperties().isEmpty()) {
+            property.setManaged(false);
+        } else {
+            StringJoiner propertyJoiner = new StringJoiner("\n");
+            seller.getProperties().stream().map(Property::toString).forEach(propertyJoiner::add);
+            property.setText(propertyJoiner.toString());
+        }
```

Comment:
Perhaps you might want to create a method for this since you will be doing this twice and its quite long?

Reply:
Changes made!